monitoring/generator.go

```diff
@@ -865,6 +865,7 @@ func main() {
 		SyntectServer(),
 		ZoektIndexServer(),
 		ZoektWebServer(),
+		SourcegraphPrometheus(),
```

Comment:
```suggestion
		Prometheus(),
```

I'd just call this Prometheus - otherwise maybe we want to call everything `Sourcegraph<Service>`? I think that's worse